src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,138 @@
+
+If you will not be using a managed migration service, see the following sections for more information on how to use SQL statements like [`IMPORT INTO`](import-into.html), [`COPY`](copy-from.html), etc.
+
+<a name="data-formats"></a>
+
+### Decide which data formats and storage media you will use
+
+It's important to decide which data formats, storage media, and database features you will use to migrate your data.
+
+Data formats that can be imported by CockroachDB include:
+
+- [SQL](schema-design-overview.html) for the [schema import](../cockroachcloud/migrations-page.html).
+- [CSV](migrate-from-csv.html) for table data.
+- [Avro](migrate-from-avro.html) for table data.
+
+The storage media you use to export / import from can be intermediate data files or streaming data coming over the network. Options include:
+
+- Local "userdata" storage for small tables (see [`cockroach userdata`](cockroach-userfile-upload.html), [Use a Local File Server](use-a-local-file-server.html)).
+- Cloud blob storage (see [`IMPORT INTO`](import-into.html), [Import Performance Best Practices](import-performance-best-practices.html), [Use Cloud Storage](use-cloud-storage.html)).
+- Direct wire transfers (see [managed migration services](#managed-migration)).
+
+Database features for export/import from the source and target databases can include:
+
+- Tools for exporting from the source database may include: `pg_dump --schema-only` and `COPY`, `mysqldump`, `expdp`, etc.
+- For import into CockroachDB, use [`COPY`](copy-from.html) or [`IMPORT INTO`](import-into.html). For a bulk data migrations, most users should use `IMPORT INTO` because the tables will be offline anyway, and `IMPORT INTO` can [perform the data import much faster](import-performance-best-practices.html) than `COPY`.
+
+Decide which of the options above will meet your requirements while resulting in a process that is [restartable](#restartable) and [scalable](#scalable).
+
+<a name="restartable"></a>
+
+### Design a restartable process
+
+To have a higher chance of success, design your data migration process so it can be stopped and restarted from an intermediate state at any time during the process. This will help minimize errors and avoid wasted effort.
+
+Keep the following requirements in mind as you design a restartable import/export process:
+
+- Bulk migrate data in manageable size batches for your source and target systems.
+    - This is a best practice. If something happens to the target cluster during import, the amount of wasted work will be minimized.
+- Implement progress/state keeping with process restart capabilities.
+- Make sure your export process is idempotent: the same input to your export process should return the same output data.
+- If possible, export and import the majority of your data before taking the source database down. This can ensure that you only have to deal with the incremental changes from your last import to complete the migration process.
```

Comment:
```suggestion
- If possible, export and import the majority of your data before taking down the legacy database. This can ensure that you only have to deal with the incremental changes from your last import to complete the migration process.
```

Reply:
took the word order rephrasing, leaving legacy alone for now as mentioned in other comment. :-)